src/runtime/Makefile

```diff
@@ -134,6 +135,8 @@ QEMUVALIDHYPERVISORPATHS := [\"$(QEMUPATH)\"]
 
 QEMUVIRTIOFSPATH := $(QEMUBINDIR)/$(QEMUVIRTIOFSCMD)
 
+QEMUNYDUSPATH := $(QEMUBINDIR)/$(QEMUNYDUSCMD)
```

Comment:
Is this `QEMUNYDUSPATH` needed? It is not used in this `Makefile` or the `configuration-qemu-nydus.toml.in`